### PR TITLE
Add changed files for account management policies

### PR DESCRIPTION
**Pull request by gomathyk**

### CODE REVIEW

File: en/docs/using-wso2-identity-server/creating-users-using-the-ask-password-option.md [hunk 1]
```diff
@@ -46,72 +44,68 @@ Follow the instructions given below to configure this feature.
     Ensure that the new Identity Listeners with
     `         orderId=95        ` and `         orderId=97        ` are set
```
Comment (sherenewso2) [inline en/docs/using-wso2-identity-server/creating-users-using-the-ask-password-option.md]: "orderId" should change to "priority" according to the new toml config

File: en/docs/using-wso2-identity-server/creating-users-using-the-ask-password-option.md [hunk 2]
```diff
@@ -46,72 +44,68 @@ Follow the instructions given below to configure this feature.
...
 
 
 Follow the steps given below to configure WSO2 IS to enable the ask
 password feature:
 
 1.  Make sure the following configuration is set (uncommented) in the
-    `           <IS_HOME>/repository/conf/identity/identity.xml          `
+    `           <IS_HOME>/repository/conf/deployment.toml          `
     file under `           <EmailVerification>          ` element below
     `           <Server>          ` element to set the redirection URL
```
Comment (sherenewso2) [inline en/docs/using-wso2-identity-server/creating-users-using-the-ask-password-option.md]: this instruction should change according to the toml config model. I think the following configs should be added into the toml file and not uncommented?

File: en/docs/using-wso2-identity-server/creating-users-using-the-ask-password-option.md [hunk 3]
```diff
@@ -46,72 +44,68 @@ Follow the instructions given below to configure this feature.
...
 
-    3.  Expand the **User Onboarding** tab and configure the **Ask
-        password code expiry time** field. Click **Update** to save
-        changes.
+        3.  Expand the **User Onboarding** tab and configure the **Ask
+            password code expiry time** field. Click **Update** to save
+            changes.
 
 2.  Optionally, if you are adding users via the management console, the
     **EnableAskPasswordAdminUI** property value needs to be added to the
```
Comment (sherenewso2) [inline en/docs/using-wso2-identity-server/creating-users-using-the-ask-password-option.md]: The "EnableAskPasswordAdminUI" property needs to change to the toml equivalent

File: en/docs/using-wso2-identity-server/creating-users-using-the-ask-password-option.md [hunk 4]
```diff
@@ -200,21 +191,20 @@ password feature:
...
-    <EmailVerification>
-            <Enable>true</Enable>
-            <LockOnCreation>true</LockOnCreation>
-            <Notification>
-                <InternallyManage>true</InternallyManage>
-            </Notification>
-        </EmailVerification>
-    ```
+    !!! info 
+        The `            EmailVerification           ` property can be
```
Comment (sherenewso2) [inline en/docs/using-wso2-identity-server/creating-users-using-the-ask-password-option.md]: change the property name according to the toml config

File: en/docs/using-wso2-identity-server/password-recovery.md [hunk 5]
```diff
@@ -38,21 +33,25 @@ notifications. The flow of this method is as follows:
 Follow the steps given below to set up and try out password recovery
 with email notification.
 
-!!! tip
-    
-    Before you begin
+!!! tip "Before you begin"
     
     Ensure that the " `         IdentityMgtEventListener        ` " with the
     `         orderId=50        ` is set to **false** and that the Identity
```
Comment (sherenewso2) [inline en/docs/using-wso2-identity-server/password-recovery.md]: the "orderId" parameter should change to "priority" and the toml configs should be added to the file

File: en/docs/using-wso2-identity-server/password-recovery.md [hunk 6]
```diff
@@ -278,49 +270,47 @@ To try this out, first create a user in the Identity Server.
...
-    Claims](https://docs.wso2.com/display/IS540/Configuring+Claims) for
-    more information on how to store the claim values in the user store.
+        -   To set up reCaptcha for password recovery with secret questions, see
+        [Configuring Google reCaptcha for Security-Question Based Password
+        Recovery](../../using-wso2-identity-server/configuring-google-recaptcha-for-security-question-based-password-recovery).
+        -   By default, the claim values of the identity claims used in this
+        feature are stored in the JDBC datasource configured in the
+        `            identity.xml           ` file. See [Configuring
+        Claims](../../using-wso2-identity-server/configuring-claims) for
+        more information on how to store the claim values in the user store.
```
Comment (sherenewso2) [inline en/docs/using-wso2-identity-server/password-recovery.md]: with the new config model, the claim values will not be in the identity.xml anymore right?

File: en/docs/using-wso2-identity-server/user-account-suspension.md [hunk 7]
```diff
@@ -23,21 +23,22 @@ users from the user store that are idle and eligible to receive a
...
 
-!!! tip
-    
-    Before you begin
+!!! tip "Before you begin"
     
     Ensure that the " `         IdentityMgtEventListener        ` " with the
     `         orderId=50        ` is set to **false** and that the "
     `         IdentityMgtEventListener        ` " with the
     `         orderId=95        ` is set to **true** in the
```
Comment (sherenewso2) [inline en/docs/using-wso2-identity-server/user-account-suspension.md]: check the instruction according to the toml config

File: en/docs/using-wso2-identity-server/username-recovery.md [hunk 8]
```diff
@@ -33,22 +33,28 @@ with the WSO2 IS.
     compatitbility and can still be used if required.
     
     
-    Before you begin
+!!! tip "Before you begin"
     
     Ensure that the " `         IdentityMgtEventListener        ` " with the
     `         orderId=50        ` is set to **false** and that the Identity
     Listeners with `         orderId=95        ` and
     `         orderId=97        ` are set to **true** in the
```
Comment (sherenewso2) [inline en/docs/using-wso2-identity-server/username-recovery.md]: change the instruction according to the toml config

File: en/docs/using-wso2-identity-server/username-recovery.md [hunk 9]
```diff
@@ -33,22 +33,28 @@ with the WSO2 IS.
...
+    enable = true
+    [event.default_listener.governance_identity_store]
+    priority= "97"
+    enable = true
+    ```
 
 
-1.  Configure the following email settings in the \<
+1.  Configure the following email settings in the <
     `           IS_HOME>/repository/conf/output-event-adapters.xml          `
```
Comment (sherenewso2) [inline en/docs/using-wso2-identity-server/username-recovery.md]: toml config?

File: en/mkdocs.yml [hunk 10]
```diff
@@ -332,23 +332,21 @@ nav:
         - 'Rule Based Provisioning': using-wso2-identity-server/rule-based-provisioning.md
         - 'Provisioning Patterns': using-wso2-identity-server/provisioning-patterns.md
       - 'Account Management Policies':
-        - 'Account Management Policies': using-wso2-identity-server/account-Management-Policies.md
```
Comment (sherenewso2) [inline en/mkdocs.yml]: why has this been removed?

File: en/mkdocs.yml [hunk 11]
```diff
@@ -332,23 +332,21 @@ nav:
...
         - 'Account Recovery':
-          - 'Account Recovery': using-wso2-identity-server/account-Recovery.md
```
Comment (sherenewso2) [inline en/mkdocs.yml]: why has this been removed?

File: en/docs/using-wso2-identity-server/username-recovery.md (hunk 9 above)
Comment (gomathyk) [reply en/docs/using-wso2-identity-server/username-recovery.md]: This has not been templated yet. Will create an issue with a few other configs.

File: en/mkdocs.yml (hunk 10 above)
Comment (gomathyk) [reply en/mkdocs.yml]: Had only the section contents. So removed the page since there's a navigation available.

File: en/mkdocs.yml (hunk 11 above)
Comment (gomathyk) [reply en/mkdocs.yml]: Had only the section contents. So removed the page since there's a navigation available.

**Pull request merged**